editor/editor_node.cpp

```diff
@@ -6926,7 +6926,7 @@ EditorNode::EditorNode() {
 	{
 		// Register importers at the beginning, so dialogs are created with the right extensions.
 		Ref<ResourceImporterTexture> import_texture;
-		import_texture.instantiate();
+		import_texture = Ref<ResourceImporterTexture>(memnew(ResourceImporterTexture(true)));
```

Comment:
I think this could be just:
```
Ref<ResourceImporterTexture> import_texture = memnew(ResourceImporterTexture(true);
```

Reply:
I wonder why the other pattern is used throughout the codebase. It seems to be always used when we assign to a pre-existing variable, though sometimes it's used in other situations as well.

If that's the reason, then I guess the idea was to have these lines consistent with others that just to `instantiate()`. Anyway, I've changed that to follow the suggestion.